RELEASES.md

```diff
@@ -1,3 +1,97 @@
+Version 1.70.0 (2023-06-01)
+==========================
+
+<a id="1.70.0-Language"></a>
+
+Language
+--------
+- [Relax ordering rules for `asm!` operands](https://github.com/rust-lang/rust/pull/105798/)
+- [Properly allow macro expanded `format_args` invocations to uses captures](https://github.com/rust-lang/rust/pull/106505/)
+- [Lint ambiguous glob re-exports](https://github.com/rust-lang/rust/pull/107880/)
+- [Initial support for return type notation (RTN)](https://github.com/rust-lang/rust/pull/109010/)
+
+<a id="1.70.0-Compiler"></a>
+
+Compiler
+--------
+- [Extend -Cdebuginfo with new options and named aliases](https://github.com/rust-lang/rust/pull/109808/)
+  This provides a smaller version of debuginfo for cases that only need line number information
+  (`-Cdebuginfo=line-tables-only`), which may eventually become the default for `-Cdebuginfo=1`.
+- [Introduce a no-op `PlaceMention` statement for `let _ =`.](https://github.com/rust-lang/rust/pull/102256/)
+  FIXME: Unclear what, if any, user-visible impact this had on landing. Clarify with T-compiler.
+- [Make `unused_allocation` lint against `Box::new` too](https://github.com/rust-lang/rust/pull/104363/)
+- [Detect uninhabited types early in const eval](https://github.com/rust-lang/rust/pull/109435/)
+- [Switch to LLD as default linker for {arm,thumb}v4t-none-eabi](https://github.com/rust-lang/rust/pull/109721/)
+
+- [Add tier 3 target for QNX Neutrino RTOS, version 7.0](https://github.com/rust-lang/rust/pull/109173/), `i586-pc-nto-qnx700`
```

Comment:
I would kindly like to request that you include the new tier 3 target loongarch64 in the release notes. Could you list it  here please? Thank you for your time and assistance. :heart: 

```
- [Add tier 3 target for LoongArch64](https://github.com/rust-lang/rust/pull/96971), `loongarch64-unknown-linux-gnu`
```